karate-tests/src/test/java/patients/healthcareWorker/patchPatient.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Patch patient - Healthcare worker access mode
+
+  Background:
+    * def utils = call read('classpath:helpers/utils.feature')
+    * def accessToken = karate.callSingle('classpath:patients/healthcareWorker/auth-redirect.feature').accessToken
+    * def requestHeaders = call read('classpath:patients/healthcareWorker/healthcare-worker-headers.js')
+    * configure headers = requestHeaders 
+    * url baseURL
+    * def nhsNumber = karate.get('nhsNumber', '9912002725')
+    * path 'Patient', nhsNumber
+    * method get
+    * status 200
+    
+    * def patientObject = response
+    * def etag = karate.response.header('etag')
+    * def originalVersion = parseInt(response.meta.versionId)
+
+  @gender
+  Scenario: Change the gender of a patient
```

Comment:
Only 1 test scenario for updates, are the rest going to be added at a later stage?

Reply:
Yes, again, the tests you see in this PR were really just basic POCs to demo karate tests rather than an attempt at extensive testing. There are lots of existing tests to port across and I'll look for additional tests too when I focus more on this.